### PR TITLE
SD-1378 show the list of packages on which the current item is included ...

### PR DESCRIPTION
**Pull request by amagdas**
...into

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by amagdas** [comment]
OK, I will make it toggle-able and hide it by default.

**Comment by amagdas** [comment]
Updated, removed the package preview, left only slugline and headline in the preview.

**Comment by amagdas** [comment]
@superdesk/developers Ping!

File: app/scripts/superdesk-authoring/packages/packages.js [hunk 1]
```diff
@@ -2,9 +2,27 @@
...
-    function PackagesCtrl($scope, superdesk) {
+    PackagesCtrl.$inject = ['$scope', 'superdesk', 'api', 'search'];
+    function PackagesCtrl($scope, superdesk, api, search) {
+        $scope.contentItems = [];
 
+        function fetchPackages() {
+            var query = search.query(null);
+            var filter = [];
+            _.forEach($scope.item.linked_in_packages, function(packageRef) {
+                filter.push({'term': {'guid': packageRef['package']}});
```
Comment (petrjasek) [inline app/scripts/superdesk-authoring/packages/packages.js]: you can use `terms` filter which is multivalue which would make this code a bit simpler

Comment (amagdas) [inline app/scripts/superdesk-authoring/packages/packages.js]: Done. Thanks, indeed it looks better :+1:

**Comment by amagdas** [comment]
@superdesk/developers Pong!

**Comment by amagdas** [comment]
@superdesk/developers Ping!!!

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**